### PR TITLE
Add pizza-wielding parrot.

### PR DESCRIPTION
**Pull request by chalisegrogan**
Because, well, pizza.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -90,6 +90,7 @@
           <li><img src="parrots/congaparrot.gif" alt="Conga Line Parrot"/> Conga Line Parrot<span tooltip="A self sufficient conga line parrot!">*</span></li>
           <li><img src="parrots/reversecongaparrot.gif" alt="Reverse Conga Line Parrot"/> Reverse Conga Line Parrot</li>
           <li><img src="parrots/partyparrot.gif" alt="Party Parrot"/> Party Parrot</li>
+          <li><img src="parrots/partyparrot.gif" alt="Pizza Parrot"/> Pizza Parrot</li>
```
Comment (antinescience) [inline index.html]: This is the same src. Shouldn't it be pizzaparrot.gif?

Comment (chalisegrogan) [inline index.html]: Good catch. 🍕

**Comment by jmhobbs** [comment]
Hey @chalisegrogan!

I merged a different commit which had a zip file update and now your PR is broken.  Could you rebase or perhaps just remove the zip file change from the commit?

Sorry!

**Comment by chalisegrogan** [comment]
@jmhobbs No problem! The zip is removed.

**Comment by jmhobbs** [comment]
Thank you!

**Pull request merged**